rpc/core/types/responses.go

```diff
@@ -145,10 +144,8 @@ type ResultDialPeers struct {
 
 // A peer
 type Peer struct {
```

Comment:
can this change be added to the upgrading.md doc.

Reply:
done!